examples/seismic/skew_self_adjoint/operators.py

```diff
@@ -0,0 +1,331 @@
+# TODO figure out if can get time sampling from the SparseTimeFuntions src/rec
+# TODO replace the conditional logic in the stencil with comprehension
```

Comment:
I don't think these should be top-level TODOs. They either go to the README, grouped with the other ones, or right above the block of code that needs modifications

Reply:
removed, they were leftovers